src/LaunchDarkly.ClientSdk/LaunchDarkly.ClientSdk.csproj

```diff
@@ -1,20 +1,23 @@
-<Project Sdk="MSBuild.Sdk.Extras">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Version>4.0.0</Version>
+    <Version>5.0.0</Version>
     <!-- The BUILDFRAMEWORKS variable allows us to override the target frameworks in
          a situation where we need to build only the .NET Standard target and do not
-         want to even mention the Android/iOS targets, because we're in an environment
-         that doesn't have the Xamarin tools installed. That is currently the case in
-         the release phase where we build HTML documentation. -->
-    <BuildFrameworks Condition="'$(BUILDFRAMEWORKS)' == ''">netstandard2.0;xamarin.ios10;monoandroid71;monoandroid80;monoandroid81</BuildFrameworks>
+         want to even mention the Android/iOS/Mac/Windows targets, because we're in an environment
+         that doesn't have the MAUI tools installed. That is currently the case in
+         the release phase where we build HTML documentation. -->     
+
+    <!-- TODO bring back netstandard target -->
+    <BuildFrameworks Condition="'$(BUILDFRAMEWORKS)' == ''">net7.0;net7.0-android;net7.0-ios;net7.0-maccatalyst;net7.0-windows</BuildFrameworks>

```

Comment:
TODO: re-add netstandard2.0 target

Reply:
Ok, well that answers a couple earlier questions maybe,

Reply:
Done